group_vars/sn07.yml

```diff
@@ -137,7 +137,11 @@ pip_install_dependencies:
   - data_hacks
   # used by reports
   - WeasyPrint
-  - total_perspective_vortex
+  - total_perspective_vortex==2.0.1
```

Comment:
This can be removed, it was fixed by [PR 145](https://github.com/usegalaxy-eu/galaxy/pull/145/files)

Reply:
This was added here based on your sn06 [commit](https://github.com/usegalaxy-eu/infrastructure-playbook/commit/25fd0d5223455ccd545cb26b0346fa0730d0dea8). It was added to keep both (sn06 and sn07) in sync. So, should this dependency itself be removed or the version pinning removed?

Reply:
```suggestion

```

Reply:
The newest version is automatically installed by galaxy so this can be removed I guess. I think we want to install possible fixes automatically